src/main/java/mat/client/export/measure/ManageMeasureExportView.java

```diff
@@ -127,6 +128,17 @@ public void setExportOptionsBasedOnVersion(String releaseVersion, boolean isComp
 		vp.add(compositeMeasurePackageRadio);
 		resetRadioButtonValues(isCompositeMeasure);
 	}
+
+	private boolean isV5OrGreater(String version) {
+		boolean result = false;
+		try {
+			result = Double.parseDouble(version.substring(1,version.length())) >= 5.0;
+		} catch (RuntimeException re) {
+		}
```

Comment:
Codacy will complain on a catch-and-swallow block. It should have a comment or a logging statement

Reply:
ok will add a comment.